gbpservice/neutron/services/grouppolicy/drivers/resource_mapping.py

```diff
@@ -139,10 +139,12 @@ class ServicePolicyPTGIpAddressMapping(model_base.BASEV2):
 
     __tablename__ = 'gpm_service_policy_ipaddress_mappings'
     service_policy_id = sa.Column(
-        sa.String(36), sa.ForeignKey('gp_network_service_policies.id'),
+        sa.String(36), sa.ForeignKey('gp_network_service_policies.id',
+                                     ondelete='CASCADE'),
         nullable=False, primary_key=True)
     policy_target_group = sa.Column(
-        sa.String(36), sa.ForeignKey('gp_policy_target_groups.id'),
+        sa.String(36), sa.ForeignKey('gp_policy_target_groups.id',
+                                     ondelete='CASCADE'),
```

Comment:
Is this a patch  that we are integrating from upstream code? Or is this a fix that we will need to upstream?